projects/eval-pqmon/src/common/pqlib_example.c

```diff
@@ -43,7 +43,7 @@
 /******************************************************************************/
 
 PQLIB_EXAMPLE pqlibExample;
-extern volatile uint8_t newSyncTimeAvailable;
+extern volatile uint8_t newSyncTimeAvailable = 0;
```

Comment:
this looks like a separate bugfix, please split it into a separate commit and document what was fixed